### ISSUE TITLE
Fix: loading state

### ISSUE THREAD

**Issue by ninadepina** [ChrisvanHvA/Appclusive#135]
falls of the page
![Screenshot 2023-06-28 at 14 58 19](https://github.com/ChrisvanHvA/Appclusive/assets/89778503/b5560666-2ea0-4782-bfc8-b1aaaa4f6740)

**Comment by noyamirai** [comment]
should be fixed on @sasjakoning 's branch "s5-wcag_check"

check to make sure its fixed! if so, u can close the issue :3

**Comment by noyamirai** [comment]
communicated via discord; its fixed!
